_data/companies_that_whiteboard.yml

```diff
@@ -562,3 +562,63 @@
   url: "https://www.makemytrip.com"
   interview_types:
   - whiteboarding
+
+- name: "RedMart"
+  glassdoor: "https://www.glassdoor.co.in/Overview/Working-at-Redmart-EI_IE928875.11,18.htm"
```

Comment:
For consistency, the Glassdoor links should point to the interview pages for a given company instead of just the overview page. In this case, it would be: https://www.glassdoor.co.in/Interview/Redmart-Interview-Questions-E928875.htm.